scripts/Packages/Package_Gullwing__QFP_SOIC_SO/ipc_gullwing_generator.py

```diff
@@ -35,7 +35,7 @@ def __init__(self, configuration):
             except yaml.YAMLError as exc:
                 print(exc)
 
-    def calcPadDetails(self, device_dimensions, ipc_data, ipc_round_base):
+    def calcPadDetails(self, device_params, device_dimensions, ipc_data, ipc_round_base):
```

Comment:
This is an unnecessary parameter here. (It holds original yaml data instead of parsed data as it should be. Meaning this function now again needs to take care of error handling.)
There must be a nicer way to achieve the same.

Reply:
Maybe it would be better to not even call this function at all if the pad size is given.